R/tags.R

```diff
@@ -492,6 +499,13 @@ tagWrite <- function(tag, textWriter, indent=0, eol = "\n") {
 
   # write attributes
   for (attrib in names(attribs)) {
+    # Can not display attrib without a key
+    if (identical(attrib, "")) {
```

Comment:
Pretty sure this would be faster? Or do we have to be considerate of 

```suggestion
    if (!nzchar(attrib)) {
```

Reply:
Using a combo of `names2(attribs)` with `attrib == ""`